src/workflows/workflow-step.ts

```diff
@@ -45,7 +45,11 @@ abstract class BaseWorkflowStepDefinition {
       {} as ManifestWorkflowStepSchema["inputs"];
 
     for (const [inputName, inputValue] of Object.entries(this.inputs)) {
-      templatizedInputs[inputName] = JSON.parse(JSON.stringify(inputValue));
+      try {
+        templatizedInputs[inputName] = JSON.parse(JSON.stringify(inputValue));
+      } catch {
+        templatizedInputs[inputName] = undefined;
```

Comment:
So if I understand correctly, setting this to `undefined` will initiate a failure mode because the backend manifest validation API will reject the manifest?

Reply:
That's how I understand it, but I haven't dug into the internals of the validation API. 

What I am observing is:

In cases where JSON.stringify returns `undefined`, JSON.parse [throws an exception](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#exceptions). 

When JSON.parse throws an exception, the Deno interpreter dies (a design decision of Deno was to [always die on uncaught exceptions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#exceptions)), bubbling the TypeScript interpreter's exit issue (the uncaught `SyntaxError` from JSON.parse) to the developer, instead of allowing the manifest validation API to do its job and return manifest validation issues to the developer.